balboa-client-dispatcher/README.md

```diff
@@ -27,7 +27,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-dispatcher-client" %% "[0.15, )",
+   "com.socrata" %% "balboa-dispatcher-client" % "0.16.0",
```

Comment:
Lets continue to use mathematical notation for patches.  This allows us to iterate on 0.16 patches.  The changes are small but usually substantial.

https://ant.apache.org/ivy/history/2.3.0/ivyfile/dependency.html#revision